packages/nimble-components/src/theme-provider/design-token-comments.ts

```diff
@@ -293,5 +293,29 @@ export const comments: { readonly [key in TokenName]: string } = {
     mentionFontFamily: 'Font family for mention views',
     mentionFontSize: 'Font size for mention views',
     mentionFontWeight: 'Font weight for mention views',
-    mentionFontLineHeight: 'Font line height for mention views'
+    mentionFontLineHeight: 'Font line height for mention views',
+    calendarStaticEventBackgroundColor:
+        'Background color for static calendar events',
+    calendarDynamicEventBackgroundColor:
+        'Background color for dynamic calendar events',
+    calendarTransientEventBackgroundColor:
+        'Background color for transient calendar events',
+    calendarStaticEventBorderColor: 'Border color for static calendar events',
+    calendarTransientEventBorderColor:
+        'Border color for transient calendar events',
+    calendarStaticEventFontColor: 'Font color for static calendar events',
+    calendarDynamicEventFontColor: 'Font color for dynamic calendar events',
+    calendarTransientEventFontColor: 'Font color for transient calendar events',
+    calendarStaticEventHoverColor:
+        'Color while hovering static calendar events',
+    calendarDynamicEventHoverColor:
+        'Color while hovering dynamic calendar events',
+    calendarTransientEventHoverColor:
+        'Color while hovering transient calendar events',
+    calendarResourceSelectionBackgroundColor:
```

Comment:
I don't know what part of a calendar is a "resource selection" can you provide a screenshot for this intended usage?

Reply:
Let's go with a less domain-specific term like "row" instead of "resource". So `calendar-row-background-color`.

Reply:
And because this is a `selected` state of a calendar-row's background, how about `calendar-row-background-selected-color`

Reply:
Updated it as calendarRowBackgroundSelectedColor - `calendar-row-background-selected-color`